fdbclient/S3BlobStore.actor.cpp

```diff
@@ -752,6 +775,10 @@ ACTOR Future<Reference<HTTP::Response>> doRequest_impl(Reference<S3BlobStoreEndp
 				bstore->setAuthHeaders(verb, resource, headers);
 			}
 
+			if (bstore->useProxy) {
+				// Has to be in absolute-form.
+				resource = "http://" + bstore->host + ":" + bstore->service + resource;
```

Comment:
Should this be conditioned to choose from `http` and `https`?

Reply:
No. https connection will be totally different. For http, we directly send normal GET/HEAD/PUT requests to proxy, except that specifying host. For https, we send a CONNECT request. I will do that in next pr.